packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/done.css

```diff
@@ -0,0 +1,48 @@
+.root {
+    height: 100%;
+}
+
+.button {
+    height: 100%;
+    width: 100%;
+}
+
+.container {
+    display: grid;
+    grid-template-rows: min-content 1fr;
+    height: 100%;
+    row-gap: 1em;
+}
+
+.contents {
+    display: grid;
+    grid-template-rows: min-content min-content;
+    margin-top: 1rem;
+    row-gap: 0.5em;
+    text-align: left;
+}
+
+.error {
+    align-self: center;
+    color: rgb(var(--venia-error));
+}
+
+.free {
+    text-transform: uppercase;
+}
+
+.heading {
+    font-weight: bold;
```

Comment:
Jimmy would ask that we use number ie `600` instead of text alias for weight :)

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2280/commits/ee446ee8c75390b705b1c031e2ed875a1685da6e.